modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -376,6 +400,33 @@ Support for the GPU instance type software stack is provided by AWS. Ensure that
 - m5zn.3xlarge (16 vCPU, 48 GiB)
 - m5zn.6xlarge (32 vCPU, 96 GiB)
 - m5zn.xlarge (4 vCPU, 16 GiB)
+- c7a.xlarge (4 vCPU, 8 GiB)
```

Comment:
Not sure if it's supposed to be alphabetical order or not - just checking if this c7a section should go after the c6 section (and before the m5 section)?

Reply:
Hi @bergerhoffer,
     Not necessarily. There is not a hard set rule for additional instance types. However, I did re-arrange that  particular category.